vignettes/wrapping-apis.Rmd

```diff
@@ -635,9 +667,11 @@ We'll use this as the basis to solve the next challenge: uploading a gist.
 
 ### Sending data
 
-To [create a gist](https://docs.github.com/en/rest/reference/gists#create-a-gist) we need to send a `POST` request with a body that contains data encodedd as JSON.
+To [create a gist](https://docs.github.com/en/rest/reference/gists#create-a-gist) we need to send a `POST` request with a body that contains data encoded as JSON.
 httr2 provides one function that does both of these things: `req_body_json()`:
 
+JENNY COMMENT HERE
```

Comment:
At first, I understood "does both of those things" to suggest that `req_body_json()` does both of these things:
* **send** a `POST` request
* with a body ...

and I was surprised that it **sent** a request. But I was just confused. It only **forms** a `POST` request. Maybe this could be reworded.